components/board/client.go

```diff
@@ -115,6 +124,14 @@ func (c *client) AnalogReaderNames() []string {
 	return copyStringSlice(c.info.analogReaderNames)
 }
 
+func (c *client) AnalogWriterNames() []string {
+	if c.getCachedStatus() == nil {
```

Comment:
If this is going to status like analog reader - we do not need it, all we need is a proper reconfigure function on board. Le'ts remove unnecessary functionality here.

Reply:
don't worry about cleaning up other functions - just don't add things to analog writer that don't need to be included!